lib/index.js

```diff
@@ -31,6 +45,57 @@ internals.implementation = function (server, options) {
                 return reply(Boom.unauthorized(null, 'Access Token'));
             }
     
+            var parts = parseCookies(authorization);
```

Comment:
@jcastets Not sure I understand, do you want to add the cookie to the response of a route handler, or earlier when the request goes through the JWT auth scheme?

I've done it in the auth scheme like this:
```
reply.state(cookieName, token, cookieOptions);
return reply.continue({ credentials: credentialsObject });
```

Careful with the scoping, since these two lines happen in the callback of `jsonwebtoken.verify`.

I don't think you need to set code 200 though, that should be the default if not specified otherwise.